draft-ietf-ppm-dap.md

```diff
@@ -1161,7 +1161,8 @@ out = VDAF.prep_next(prep_state, None)
 ~~~
 
 `vdaf_verify_key` is the VDAF verification key shared by the aggregators;
-`agg_id` is the aggregator ID (`0x00` for the Leader and `0x01` for the helper);
+`agg_id` is the aggregator ID, equal to the role in {{protocol-definition}}
+(`0x02` for the Leader and `0x03` for the helper);
```

Comment:
This is a breaking change for VDAF. We could either:
1. Update the agg ID semantics in VDAF and cut a draft
2. Align server role with agg ID

Option (2.) might look like this:

```suggestion
`agg_id` is the aggregator ID, equal to `server_role - 0x02`, where `server_role` is the aggregators' role in {{protocol-definition}};
```
Alternatively, we can re-assign the server roles to align with the VDAF draft.

Reply:
Duplicates @divergentdave's comment.